src/.env.sample

```
@@ -286,3 +286,6 @@ SURVEY_SERVICE_BASE_URL=/survey/
 ENTITY_MANAGEMENT_SERVICE_NAME=/entity-management/
 #tenant event from user service
 USER_SERVICE_TENANT_CREATION_TOPIC='dev.tenantEvent'
+
+# org Extension create Topic from userSevice
+ORGANIZATION_UPDATES_TOPIC="dev.organizationEvent"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)